BuildGenerated.sh

```diff
@@ -2,6 +2,15 @@
 
 set -e
 
+# Try to detect Python 2. It's quite different between Windows and Linux.
+if which python > /dev/null && python --version 2>&1 | grep -q "Python 2"; then declare -r PYTHON2=python
```

Comment:
I'm never certain wether this needs to be in quotations or not, flagging just in case:
```suggestion
if which python > /dev/null && python --version 2>&1 | grep -q "Python 2"; then declare -r PYTHON2="python"
```
And if it does need the quotations, then the one two lines down needs them as well.

Reply:
Nope, I don't believe it does. (This code was taken from google-cloud-dotnet, which has been working fine :)